packages/core/aqua-to-js/src/generate/__test__/generate.spec.ts

```diff
@@ -40,12 +40,15 @@ describe("Aqua to js/ts compiler", () => {
       },
     };
 
+    // @ts-expect-error don't use compileFromPath directly here
```

Comment:
don't really understand what you are trying to say here

Reply:
compileFromPath - internal api from aqua-to-js package. This comment suggests to use exposed api of aqua-to-js instead of internal api.

Reply:
compileFromPath is aqua-api not aqua-to-js right? I am super confused

I think the most correct thing to do here is to use the type that aqua-api returns in aqua-to-js instead of using interfaces package in aqua-to-js

since @fluencelabs/aqua-api is installed as dev dependency for this package - you can easilly use types from it to type everything that this package accepts e.g. extract type like this Awaited<ReturnType<typeof compileFromPath>>

Reply:
Yes, i said nonsense above. confused it with js-client package.

Applying your changes not so simple.
It required thorough code analysis and refactoring.
`aqua-to-js` uses newer interface package, `aqua-api` uses older and they diverged.
newer interface package has more presice types that cannot accept older ones.

I will refactor it in another PR. Need to discuss usage of interface package with aqua team also.

Reply:
Added TODO additonally to comments